### PR TITLE
CLDC-3615 Add pending email change banner

### PR DESCRIPTION
**Pull request by kosiakkatrina**
image

### CODE REVIEW

File: app/helpers/user_helper.rb [hunk 1]
```diff
@@ -14,4 +14,23 @@ def can_edit_org?(current_user)
...
+    if current_user == user
+      "You have requested to change your email address to #{user.unconfirmed_email}."
+    else
+      "There has been a request to change this user’s email address to #{user.unconfirmed_email}."
+    end
+  end
+
+  def pending_email_change_banner_text(current_user)
+    text = "A confirmation link has been sent to the new email address. The current email will continue to work until the change is confirmed."
+    text += " Deactivating this user will cancel the email change request." if current_user.support?
```
Comment (RachaelBooth) [inline app/helpers/user_helper.rb]: Can coordinators deactivate users in their org? If so, they should probably see this too.

File: spec/helpers/user_helper_spec.rb [hunk 2]
```diff
@@ -61,4 +61,65 @@
...
+  describe "display_pending_email_change_banner?" do
+    context "when the user doesn't have an unconfirmed email" do
+      let(:user) { FactoryBot.create(:user, :data_provider, unconfirmed_email: nil) }
+
+      it "does not display pending email change banner" do
+        expect(display_pending_email_change_banner?(user)).to be false
+      end
+    end
+
+    context "when the user doesn't has the same unconfirmed email as current email" do
```
Comment (RachaelBooth) [inline spec/helpers/user_helper_spec.rb]: You've got a stray "doesn't" in this context name, and in the context below.

File: app/helpers/user_helper.rb (hunk 1 above)
Comment (kosiakkatrina) [reply app/helpers/user_helper.rb]: Yes they can, and I agree, makes sense to show it to coordinators as well!

File: spec/helpers/user_helper_spec.rb (hunk 2 above)
Comment (kosiakkatrina) [reply spec/helpers/user_helper_spec.rb]: That's what I get for copying my test cases.. Thank you!

**Review by Dinssa — APPROVED** [review]

**Pull request merged**